mcs/swagger.yml

```
@@ -846,3 +884,40 @@ definitions:
   # Structure that holds the `Bearer {TOKEN}` present on authenticated requests
   principal:
     type: string
+  startProfilingItem:
+    type: object
+    properties:
+      nodeName:
+        type: string
+      Success:
```

could you please change this lower case?
```suggestion
      success:
```